lib/documents/schemas/utaac_decisions.json

```diff
@@ -322,6 +322,10 @@
           "label": "Bereavement and death benefits - bereaved parents allowance",
           "value": "bereavement-and-death-benefits-bereaved-parents-allowance"
         },
+        {
+          "label": "Bereavement and death benefits - bereavement support payment",
+          "value": "bereavement-and-death-benefits-bereavement-support-payment"
+        },
         {
           "label": "Bereavement and death benefits - bereavement payments",
           "value": "bereavement-and-death-benefits-bereavement-payments"
```

Comment:
Is "Bereavement and death benefits - bereavement support payment" different from "Bereavement and death benefits - bereavement payments"?

Reply:
Yeah, it does seem to be: https://www.gov.uk/bereavement-payment and https://www.gov.uk/bereavement-support-payment

Reply:
👍